### DISCUSSION TITLE
Guidelines for submiting a proposal to GSoC 2024 for RUXAILAB (aka Uramaki LAB)

### DISCUSSION THREAD

**Discussion by KarinePistili** [ruxailab/RUXAILAB#373] [Announcements]
### RUXAILAB (aka Uramaki LAB) has been accepted as a mentor organization for Google Summer of Code for the first time this year! :tada: 

We are very excited for this new journey on our community and so thrilled to have many of you new contributors joining us! We appreciate your work and enthusiasm.

For that, we want to be as clear and transparent with the guidelines of proposal submission to help you write strong proposals and have more chances to be selected.

#### 1. Read the GSoC 2024 information page
There we have multiple information to help you get started as well the Ideas List and Proposal Template. Refer to: https://github.com/ruxailab/gsoc

#### 2. Check out the repositories related and what fits your interests

We suggest you to carefully run and analyse them, ask questions on the discussion and interact with the developers to increase your knowledge about the organization.

Refer to the following discussions:
* [Issues to get started contributing](https://github.com/orgs/ruxailab/discussions/352)
* [Introduce Yourself (GSoC 2024)](https://github.com/orgs/ruxailab/discussions/306)

#### 3. Seek feedback on your proposals early in the process

Reach out to developers and mentors of RUXAILAB in the [Discussions](https://github.com/orgs/ruxailab/discussions) or join our [Discord server](https://discord.gg/YnkDk9BNYK) to get feedback about your ideas and work to improve it.

#### 4. Contribute to the project :computer: 

Contributing with the different projects helps us to get to know you and your work, so being interactive will give you points.

#### 5. Don't be shy. Talk to us :wink: 

We would love to get to know all of you and we are trying to interact with as many we can. We only ask for some patience if we take some time answering, but I assure we are doing our best.

#### 6. Be attentive

Always be attentive to the annoucements.

### Good luck! May the force be with you! :sparkles:

**Comment by KarinePistili** [comment]
Hello Contributors!

Thank you so much for the appreciation and interest for our small organization. We are so happy to see so many faces engaging with us.

Don't forget to submit your proposals on GSoC dashboard, as there is official way to submit and be able to get selected.

GSoC Dashboard: https://summerofcode.withgoogle.com/

Tomorrow is the last day to submit.

Good luck to you all! 🚀 💪

**Comment by KarinePistili** [comment]
Thank you for all the proposal submissions! We are reviewing all of them carrefully. Good luck to you all!

**Comment by akash8843** [comment]
Hello @KarinePistili, 
My name is Akash Shukla, and I’m an aspiring software engineer passionate about web development. I have hands-on experience with HTML, CSS, JavaScript, Bootstrap, and Node.js, and I’ve been actively learning through real-world projects. 

I’m really interested in contributing to the RUXAILAB project for GSOC 2025. However, I feel like I might be a bit late in starting my contributions, and I’m worried about catching up. 

Could you please guide me on how I can still contribute and where I should begin? I’m eager to learn, work hard, and make meaningful contributions to the project with your support. 

Looking forward to your response! 

Best,  
Akash Shukla

**Comment by ishachadalavada** [comment]
Hi @KarinePistili,
My name is Isha Chadalavada, and I am a current undergraduate senior at Northeastern, studying computer science. I have experience with machine learning and artificial intelligence research with projects at the Burnes Center and Center on Crime, Race and Justice at Northeastern, specifically using python, tensorflow APIs, and ChatGPT/Llama API.

I would like to be a contributor to the RUXAILAB and would love to get in touch about contributing to the AI heuristics project. I was hoping for more information about how to get feedback about my proposal and learn if there is anything additional I could do to stand out.

Thanks,
Isha Chadalavada
